### ISSUE TITLE
Experimental issues

### ISSUE THREAD

**Issue by lianxiliangnianbandeyuan** [Abh4git/TrafficScheduling#3]
How did you get the configuration inside omnet++? I see that the genetic algorithm inside the PDF gets the result of when the stream scheduling is done right?

**Comment by Abh4git** [comment]
Configuration is translated to the JSON file in OMNET. It will look something like this. 
![image](https://github.com/user-attachments/assets/6c9ce9f7-ce77-4e30-8ca5-6826618991db)
The time details from the schedule is translated into milliseconds and placed here in each tsn switch 
![image](https://github.com/user-attachments/assets/4f4d76da-423e-4f5b-a931-d68e8408c0ad)

**Comment by Abh4git** [comment]
Slots are timeslots for TSN. This JSON file is used by TSN configurator in OMNET++. You need to comment some parts of the configurator after overriding existing example:
![image](https://github.com/user-attachments/assets/9a825ff0-8f20-44fc-ad1a-310001882979)
1. Comment executeTSNsched . So it wont create a new file. 
2. Modify the filename in Output to point to your JSON file (modified based on schedule).

**Comment by lianxiliangnianbandeyuan** [comment]
The omnet++ program here is a rewritten program by you, right? Is it irrelevant to the solution of this problem using genetic algorithms?

**Comment by Abh4git** [comment]
All cases, either you should change the TSN Scheduler Code or make your adaptations
1. See this is using the tsn defined dumbell network. So you cannot go and create a topology you like. 
![image](https://github.com/user-attachments/assets/b5baf988-7ede-4c65-afc2-93493b6fd7ab)
![image](https://github.com/user-attachments/assets/a3293545-e030-480a-a9ba-d07a9998ea6d)
![image](https://github.com/user-attachments/assets/a8d3995b-aed6-44c0-b879-342e6f169a6d)
2.  In order to ensure the file used the TSNScheduleConfigurator is the new JSON, we need to stop running the java  file and then take this as input (Output from java)
Basically I wanted to customize the topology and be able to run my scenarios. Then it is required.

**Comment by lianxiliangnianbandeyuan** [comment]
Thank you for your answer.
